compiler/noirc_frontend/src/elaborator/comptime.rs

```diff
@@ -345,12 +345,15 @@ impl<'context> Elaborator<'context> {
         }
     }
 
-    pub fn setup_interpreter<'local>(&'local mut self) -> Interpreter<'local, 'context> {
+    pub fn setup_interpreter<'local>(
+        &'local mut self,
+        location: Location,
+    ) -> Interpreter<'local, 'context> {
```

Comment:
I was thinking we'd store the full call stack in the elaborator so that when we go `interpreter -> elaborator -> interpreter` it isn't lost. I expect this case to be rare though so I won't block on it.